app/src/main/java/com/kwartracker/android/home/ui/DateRangeDialog.kt

```diff
@@ -0,0 +1,55 @@
+package com.kwartracker.android.home.ui
+
+import android.app.Dialog
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import androidx.databinding.DataBindingUtil
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.kwartracker.android.R
+import com.kwartracker.android.databinding.DialogDateRangeHomeBinding
+
+class DateRangeDialog : BottomSheetDialogFragment() {
+    private lateinit var binding: DialogDateRangeHomeBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = DataBindingUtil.inflate(
+            inflater,
+            R.layout.dialog_date_range_home,
+            container,
+            false
+        )
+
+        return binding.root
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val dialog = super.onCreateDialog(savedInstanceState)
+        dialog.window?.setBackgroundDrawableResource(
+            R.color.transparent_white
+        )
+        dialog.window?.setDimAmount(0F)
+
+        return dialog
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        dialog?.window?.setLayout(
+            WindowManager.LayoutParams.MATCH_PARENT,
+            WindowManager.LayoutParams.MATCH_PARENT
+        )
+
```

Comment:
@danacealfonso 
```suggestion
```
These lines are unnecessary since the default layout parameter of Window is MATCH_PARENT.

Reply:
remove in 77892c11efb4c66a48f1375cfe73b3ccb564b7e6